renderdoc/driver/vulkan/vk_shader_feedback.cpp

```diff
@@ -1584,7 +1600,7 @@ void VulkanReplay::FetchShaderFeedback(uint32_t eventId)
 
     AnnotateShader(*pipeInfo.shaders[5].refl, *pipeInfo.shaders[5].patchData,
                    ShaderStage(StageIndex(stage.stage)), stage.pName, offsetMap, maxSlot, false,
-                   bufferAddress, useBufferAddressKHR, modSpirv, printfData[5]);
+                   bufferAddress, useBufferAddressKHR, false, modSpirv, printfData[5]);
```

Comment:
I assume it doesn't make sense for a multiview vertex shader to not declare and use the capability, so you don't ever need to "force" it on for the vertex shader?

Reply:
This AnnotateShader is the compute one (result.compute), on the VS side I do force it to true as well
```
  if(usesMultiview && (stage == ShaderStage::Pixel || stage == ShaderStage::Vertex))
  {
    editor.AddCapability(rdcspv::Capability::MultiView);
    editor.AddExtension("SPV_KHR_multiview");
  }

```